raiden/network/rpc/client.py

```diff
@@ -509,18 +509,18 @@ def __init__(
 
         result = jsonrpc_client.call(
             'eth_getCode',
-            address_encoder(token_address),
+            token_address,
```

Comment:
I follwed this https://github.com/ethereum/wiki/wiki/JSON-RPC#eth_getcode and accordingly I get the correct result. Something like this.
```
[12] > /home/krishna/raidenforked/raiden/raiden/network/rpc/client.py(518)__init__()
-> if result == '0x':
(Pdb++) n
[12] > /home/krishna/raidenforked/raiden/raiden/network/rpc/client.py(523)__init__()
-> proxy = jsonrpc_client.new_abi_contract(
(Pdb++) result
u'0x60606040523615604f5760e060020a6000350463095ea7b38114605357806318160ddd14606857806323b872dd14608157806370a08231146068578063a9059cbb146053578063dd62ed3e14609c575b6002565b3460025760886004356024355b600092915050565b3460025760005b60408051918252519081900360200190f35b3460025760005b604080519115158252519081900360200190f35b34600257606f600435602435606056'

```